package.json

```diff
@@ -9,11 +9,11 @@
     "test": "./node_modules/mocha/bin/mocha --require ./tests/utils/bootstrap.js --full-trace --recursive -R mocha-unfunk-reporter tests/",
     "test-watch": "npm run lint && ./node_modules/mocha/bin/mocha --require ./tests/utils/bootstrap.js --full-trace --watch --recursive -R mocha-unfunk-reporter tests/",
     "coverage": "./node_modules/.bin/istanbul cover --include-all-sources true -x **/examples/**/* --print both ./node_modules/.bin/_mocha -- --require ./tests/utils/bootstrap.js -R spec --recursive tests/",
-    "commit": "git-cz",
-    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -w -r 0",
-    "precommit": "npm run lint && npm test",
-    "prepush": "npm run lint && npm test",
-    "commitmsg": "validate-commit-msg",
+    "commit": "npm --version",
```

Comment:
Seems like an error to change all this scripts

Reply:
I must have forgotten to put them back.

I've removed them because making commits with Husky was becoming a nightmare.

Reply:
well if you commit often I understand, but at the end I find it useful for running test automatically & checking my commit messages